EmployeesSalaries/EmployeesSalaries/Models/Interfaces/IEmployeeSalaryCalculater.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace EmployeesSalaries.Models.Interfaces
+{
+    public interface IEmployeeSalaryCalculater
+    {
+        public double BasicSalary { get; set; }
+        public double GetTotalSalary();
+        public bool IsRequiredCalculater(string Role);
```

Comment:
refactor your code to remove IsRequiredCalculater method. we don't need this

Reply:
Removed